### ISSUE TITLE
Compatibility with RHS: Escalation

### ISSUE THREAD

**Issue by jonpas** [KoffeinFlummi/AGM#1454]
- [ ] RHS' Abrams FCS doesn't work.
- [x] M2A3 Bradley has no main gun.
- [x] BMP/BMD IFC + T80 Tank: if shoot out main shells (loaded by default in main gun) you can't reload other shells (no action menu item) 
- [ ] The machine gunners need spare barrels
- [ ] Trained pilots need coefficients to withstand g forces
- [ ] Non steerable parachutes for jet pilots
- [ ] Earplugs for some
- [ ] Equippment for pioneers

---

RHS: Escalation actually has visible wheel on BMPs (and possible something else), that you can transfer from one to another. Is there any chance this could be integrated with AGM's Logistics and AGM's Spare Tires?

**Comment by kripto202** [comment]
I noticed that the sniper scopes for escalation doesn't work with agm.

**Comment by commy2** [comment]
Yes. They aren't compatible, but that requires some deeper setup.

**Comment by jonpas** [comment]
Abrams FCS seems to work, but pops up an error that's RHS's, distance and all is good though
(http://feedback.rhsmods.org/view.php?id=161).

**Comment by Trigger81** [comment]
Abrams FCS dosen´t work with AGM 0.94.3 and RHS 0.3.6
